apps/cognito-functions/src/templates/confirmation-message.ts

```diff
@@ -0,0 +1,80 @@
+import { COMPANY_LOGO } from './company-logo';
+
+const TRANSLATIONS: { readonly [k: string]: string } = {
```

Comment:
We should think about e-mail translations and replace this object with something more appropriate